src/ressources/user/__init__.py

```diff
@@ -1,3 +1,3 @@
 from .UserAdd import UserAdd
 from .UserGet import UserGet
-from .UserGetNotif import UserGetNotif
+from .UserPref import UserPref
```

Comment:
import UserGetPref ?

Reply:
Actually UserPref is correct, because I want to implement two methods : POST et GET, for now POST isn't implemented. That's the logic behind the "Ressource"